src/popover/popover.js

```diff
@@ -399,6 +399,16 @@ class Popover extends React.Component<PopoverPropsT, PopoverPrivateStateT> {
     return childArray[0];
   }
 
+  getMountNode(): HTMLElement {
+    const {mountNode} = this.props;
+    if (mountNode) {
+      return mountNode;
+    }
+    // Flow thinks body could be null (cast through any)
+    // eslint-disable-next-line flowtype/no-weak-types
+    return ((document.body: any): HTMLBodyElement);
```

Comment:
Need to wrap it in the `if (__BROWSER__) {}`

Reply:
I'm not sure because I get this implementation from the Modal component and I can't see the condition there.

Reply:
It is already checked on `render`.